app/services/discord/eval.js

```diff
@@ -0,0 +1,8 @@
+import Discord from 'discord.js';
+
+export default async function d_eval(message, opt) {
+
+    const out = eval(opt)
```

Comment:
`eval` in Javascript is typically considered to be a major security issue, as it allows someone from Discord to run arbitrary code on the server in question. Since this server environment runs many other apps (some with sensitive information), this could be disastrous if merged. Note the banner here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval. As we say in the JS community "ev_al_ is ev_il_".